.github/renovate.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Moved the config file into the .github folder as supported: https://docs.renovatebot.com/configuration-options/